packages-new/components/flag/src/components/osds-flag/constants/flag-iso-code.ts

```diff
@@ -0,0 +1,18 @@
+import { ODS_COUNTRY_ISO_CODE } from "@ovhcloud/ods-common-core";
+
+const ODS_FLAG_ISO_CODE = {
+    ...ODS_COUNTRY_ISO_CODE,   
+    EU: 'eu', // Europe
+    UN: 'un', // United Nations
+    XK: 'xk', // Kosovo
+} as const;
```

Comment:
Object.freeze break the typage

Reply:
For history purpose after solving it together: issue is with typescript version < 4.7.*
Solution is to enforce IDE to use the workspace version